### DISCUSSION TITLE
Screencast: Modern Java: Download and Upload Files from SFTP Servers from Java using the sshj library

### DISCUSSION THREAD

**Discussion by productivity-for-programmers** [hierynomus/sshj#836] [Show and tell]
Thanks for creating such a great library. It's great to have an alternative to jsch.

I did a screencast on this library around a year ago:

https://www.youtube.com/watch?v=ySCcZa9Y-Zw

Hope that helps some people getting started with this library.

**Comment by ashishiter** [comment]
How can i add a certificate file along with the private key which will be used by the target server . The scp command look like this :"scp -i id-rsa -o "CertificateFile id-rsa-cert.pub" -o "ProxyCommand ssh -qx -i id-rsa -o"CertificateFile id-rsa-cert.pub" svcAutobahn@jump.mmm.com -W %h:%p" foo.bar ec2-user@targetHost:"
